.github/policies/resourceManagement.yml

```
@@ -24,6 +24,22 @@ configuration:
       - closeIssue
       - addReply:
           reply: "Closing this issue as we've seen no reply to the request for more information. If you are able to get the requested information, please add it to the issue and we will retriage it. "
+   
+    - description: Close stale PRs
+      frequencies:
+      - daily:
+          time: 12:00
+      filters:
+      - isOpen
+      - isPullRequest
+      - not:
+          isDraft

```

Should this be an array element?

```suggestion
          - isDraft
```